### ISSUE TITLE
declaration specifier expected near 'ssize_t' at line 2

### ISSUE THREAD

**Issue by ghost** [Wiladams/lj2procfs#2]
Hello there,

This is my current environment/situation:
-  Fedora 24 Desktop up to date (with the proper c libs)
-  Using luaver to manage Lua, LuaJIT and luarocks versions (5.1 and 5.3.3; 2.0.4; and 2.3.0, respectively)
-  Installed lj2procfs through luaver's download of luarocks (Fedora 24 versions does not find lj2procfs. I don't know why). Which installs it in .luaver respective directory in your user's home.
-  When running a lua code with LuaJIT I get title as an error. Easily fixed by patching the libc.lua file with the most current update.

I don't know if your luarocks repo is up to date (nor do I know how it works behind the curtains, but doing this for this particular situation fixed my problem). Could you please update your luarocks repo?

**Comment by Wiladams** [comment]
Yes, it's true.  There was a change in the LuaJIT 2.1 beta to add the ssize_t type.  I'll change the code to be conditional on its presence.
